bin/cli.js

```diff
@@ -237,12 +259,24 @@ program
     "Try to create all export files stored in the repository"
   )
   .action((options) => {
-    cliUtils.checkUniqueOption(["name", "all"], options);
-    cliUtils.checkDefaultFirm(options.firm, firmIdDefault);
+    const settings = runCommandChecks(
```

Comment:
Should we have checkPartnerSupport? It is in all other create commands

```suggestion
    cliUtils.checkPartnerSupport(options);
    const settings = runCommandChecks(
```